src/newlines.h

```diff
@@ -35,4 +35,6 @@ chunk_t *newline_force_after(chunk_t *pc);
 void newline_del_between(chunk_t *start, chunk_t *end);
 chunk_t *newline_add_between(chunk_t *start, chunk_t *end);
 
+int newlines_between(chunk_t *pcS, chunk_t *pcE, scope_e scope = scope_e::ALL);
```

Comment:
Yes you are right I really should had add some documentation for the new code, I'll make a PR for that.